release/README.md

```diff
@@ -2,8 +2,37 @@
 
 This directory contains tools for Kata Containers releases.
 
+## update-repository-version.sh ##
+
+This script is used to create a Github pull request (a.k.a PR) to change version in all
```

Comment:
Suggested rewrite: This script creates a GitHub pull request (a.k.a PR) to change the version in all